wireless_msgs/msg/Connection.msg

```diff
@@ -11,3 +11,4 @@ int16 noise_level  # dBm
 
 string essid
 string bssid
+float64 frequency
```

Comment:
I'm not sure. Does scanning continue while we're rolling around? If not, we may not have information on all the networks.